lib/SILOptimizer/PassManager/PassManager.cpp

```diff
@@ -1929,7 +1929,11 @@ OptionalBridgedFunction BridgedPassContext::lookupStdlibFunction(BridgedStringRe
 
   SILDeclRef declRef(decl, SILDeclRef::Kind::Func);
   SILOptFunctionBuilder funcBuilder(*invocation->getTransform());
-  return {funcBuilder.getOrCreateFunction(SILLocation(decl), declRef, NotForDefinition)};
+  SILFunction *function = funcBuilder.getOrCreateFunction(SILLocation(decl), declRef, NotForDefinition);
```

Comment:
It's not good to hide such kind of logic in a bridging function.
Instead it's better to explicitly call `Context.loadFunction` at the call site (in the `optimize` function in this case).